libcudacxx/test/public_headers/CMakeLists.txt

```
@@ -46,6 +46,7 @@ function(libcudacxx_add_public_header_test header)
                          PRIVATE
                          ${headertest_warning_levels_device}
                          -DLIBCUDACXX_ENABLE_EXPERIMENTAL_MEMORY_RESOURCE)
+  target_compile_definitions(headertest_${header_name} PRIVATE _CCCL_HEADER_TEST)
   target_compile_definitions(headertest_${header_name} PRIVATE CCCL_ENABLE_ASSERTIONS)
   target_compile_definitions(headertest_${header_name} PRIVATE CCCL_IGNORE_DEPRECATED_CPP_DIALECT)
   target_compile_definitions(headertest_${header_name} PRIVATE CCCL_ENABLE_OPTIONAL_REF)
```

Can we collapse those into a single call to 
```suggestion
  target_compile_definitions(headertest_${header_name} PRIVATE 
    _CCCL_HEADER_TEST
    CCCL_ENABLE_ASSERTIONS 
    CCCL_IGNORE_DEPRECATED_CPP_DIALECT
    CCCL_ENABLE_OPTIONAL_REF)
```

---

@alliepiper for cmake knowledge

---

According to the docs [docs](https://cmake.org/cmake/help/latest/command/target_compile_definitions.html) this should be fine